res/controllers/Denon-MC7000-scripts.js

```diff
@@ -744,25 +845,25 @@ MC7000.sortLibrary = function(channel, control, value) {
 /* LEDs for VuMeter */
 // VuMeters only for Channel 1-4 / Master is on Hardware
 MC7000.VuMeter = function(value, group) {
-    var VULevelOutValue = engine.getValue(group, "PeakIndicator") ? MC7000.VuMeterLEDPeakValue : value*value*value*value*0x69,
+    var VULevelOutValue = engine.getValue(group, "PeakIndicator") ? MC7000.VuMeterLEDPeakValue : Math.pow(value, 4) * MC7000.VuMeterLEDPeakValue - 1,
         deckNumber = script.deckFromGroup(group);
-
     midi.sendShortMsg(0xB0 + deckNumber - 1, 0x1F, VULevelOutValue);
 };
 
 /* LEDs around Jog wheel */
 MC7000.JogLed = function(value, group) {
-    var deckNumber = script.deckFromGroup(group);
     // do nothing before track starts
-    if (value < 0) return;
-
-    var trackDuration = engine.getValue(group, "duration"),
+    if (value === 0) {
+        return;
+    }
+    var deckNumber = script.deckFromGroup(group),
+        trackDuration = engine.getValue(group, "duration"),
         position = value * trackDuration / 60 * MC7000.scratchParams.recordSpeed,
         // LED ring contains 48 segments with each LED activated by the next even number
         LEDmidiSignal = 48 * 2,
```

Comment:
same here, please don't start variable names with a capital letter.

Reply:
Also here - comes from your team mate exactly like this. I did not even have the variable there calculating 48*2 but used 96 inside the calculation below. now this team mate suggestion is wrong again?